### ISSUE TITLE
Scrolling past a chart doesn't work

### ISSUE THREAD

**Issue by benoj** [reactivemarkets/react-financial-charts#525]
#### I'm submitting a...

- [x] bug
- [x] question
- [ ] feature
- [ ] chore

#### What is the current behavior

 Currently when we have a chart in the middle of a scrollable page (and zoom is disabled), when a user scrolls down the page and the mouse enters over a chart, then the scroll stops and is stuck over the chart (presumably because it is trying to capture the zoom scroll)

#### What is the expected behavior

The user should be able to scroll past the chart without being stuck on it. Is this possible already and we just don't have the right settings configured?

#### Other information

```

  <ChartCanvas
        height={gridHeight}
        ratio={ratio}
        width={width}
        margin={margin}
        data={data}
        displayXAccessor={displayXAccessor}
        seriesName="Data"
        xScale={xScale}
        xAccessor={xAccessor}
        xExtents={xExtents}
        disableZoom={true}
        disableInteraction={false}
        disablePan={true}
        plotFull={true}
      >
        <Chart id={1} yExtents={yExtents} yPanEnabled={false}>
          <AreaSeries yAccessor={yAccessor} {...rest} connectNulls={true} />
          <XAxis
            zoomEnabled={false}
            fontFamily="'Roobert TRIAL', 'sans-serif'"
            fontSize={12}
            showDomain={false}
            showTicks={false}
          />
          <YAxis
            zoomEnabled={false}
            showGridLines={showGridLines}
            gridLinesStrokeWidth={1}
            ticks={4}
            fontSize={12}
            showDomain={false}
            showTicks={false}
            tickFormat={pricesDisplayFormat}
            axisAt="left"
            orient="left"
          />
          <HoverTooltip
            background={{
              fillStyle: 'transparent',
            }}
            yAccessor={yAccessor}
            toolTipFillStyle="#000000"
            toolTipStrokeStyle="#ffffff"
            fontFill="#ffffff"
            fontSize={18}
            origin={(
              props: HoverTooltipProps,
              moreProps: any,
              bgSize: { width: number; height: number },
              pointWidth: number
            ) => {
              const verticalPadding = 4
              const { yAccessor } = props
              const { xAccessor, currentItem, width, chartConfig } = moreProps
              const xValue = xAccessor(currentItem)
              const yValue = yAccessor(currentItem)

              const normalizeX = (x: number) => {
                return x < width / 2 ? x : x - bgSize.width
              }

              const normalizeY = (y: number) => {
                return y - bgSize.height <= 0
                  ? y + verticalPadding
                  : y - bgSize.height - verticalPadding
              }

              const x = normalizeX(xScale(xValue) * pointWidth)
              const y = normalizeY(chartConfig.yScale(yValue))

              return [x, y]
            }}
            tooltip={{
              content: ({ currentItem, xAccessor }) => ({
                x: pricesDisplayFormat(currentItem.price.amount || 0),
                y: [
                  {
                    label: '',
                    value: dateFormat(xAccessor(currentItem)),
                  },
                ],
              }),
            }}
          />
        </Chart>
      </ChartCanvas>
```

**Comment by markmcdowell** [comment]
`disableInteraction` will do this and ignore the scroll. Although I do agree if you disable zoom you wouldn't expect it to still capture.

**Comment by benoj** [comment]
Cool -  I just tried the `disableInteraction` and that fixes the scroll past behaviour, however, we also use tooltips and unfortunately this prevents them from working too.

One hack I was thinking was to enable interaction on hover for a period of time (e.g. user hover's the chart for 2 second) - but this feels perhaps a bit of a hack. Is there another supported way of enabling the user to scroll past but also have the tooltips available?

**Comment by benoj** [comment]
I suspect that if we added an additional `disableWheel` option around this - then that should work.
https://github.com/reactivemarkets/react-financial-charts/blob/f5a2ab4b537652770c4ff8a1805678fb149861e9/packages/core/src/EventCapture.tsx#L146

I also think _think_ that disable pan + disable zoom should have this same effect - but it seems like it doesn't right?

**Comment by benoj** [comment]
Opened a PR to resolve this without resorting to a hack on my end :)

**Comment by markmcdowell** [comment]
Published in [v1.3.1](https://github.com/reactivemarkets/react-financial-charts/releases/tag/v1.3.1)
